Initial lang - csv

xTooth opened the pull request:
closes Ohtu-FaceTed/FaceTed-Search#109
closes Ohtu-FaceTed/FaceTed-Search#110 
closes Ohtu-FaceTed/FaceTed-Search#111  
also includes 2/3 of Ohtu-FaceTed/FaceTed-Search#108.

xTooth committed:
Update attributes.csv

MODIFIED
> data/attributes.csv
- now contains tooltip and language support.

kotommi committed:
modify gitignore

kotommi committed:
added json language strings

xTooth committed:
Merge branch 'Lang-csv' of https://github.com/Ohtu-FaceTed/FaceTed-Backend into Lang-csv

xTooth committed:
Renamed new csv.

xTooth committed:
Commit for debugging pourposes

kotommi committed:
remove index column from csv

kotommi committed:
update gitignore

kotommi committed:
load buildingdata correctly

kotommi committed:
remove debug prints

kotommi committed:
add field for tooltip

kotommi committed:
Add attribute tooltips to db

xTooth committed:
Update attribute_groups.csv

MODIFIED:
>attribute_groups.csv
-now contains languagesupport for mutiquestions.

xTooth committed:
Test fixes

MODIFED
>src init
-removed data point to fix tests.
>tests init
- added tooltip attribute to testdata

xTooth committed:
Pylint

Ran lint. No "real" changes.

xTooth committed:
csv attribute name multilanguage support.

MODIFIED
>attributes.csv
- added multilanguage tags to attribute_name
ADDED
>script for editing csv columns. called csve.py

xTooth committed:
Update attribute_groups.csv

just fixed a sentence that was bugging me.

EssKayz commented on src/building_data.py:
```diff
@@ -18,30 +19,32 @@
                                    'group_id': [None, None, None, '1', '1'],
-                                   'active': [True, True, True, True, True]})
+                                   'active': [True, True, True, True, True],
+                                   'attribute_tooltip': ['Onko rakennuksessa asunnot?',
```
Should attributes have the translations also in the default_attributes list for both question and tooltip?

EssKayz commented on tests/__init__.py:
```diff
@@ -19,27 +19,32 @@ def init_test_db(app):
                                  attribute_name='Asunnot',
                                  attribute_question='Onko rakennuksessa asunnot?',
```
Shoudl attribute_question get the translations for tests too, or do we just test with one language?

xTooth committed:
Update __init__.py

MODIFIED
>test/__init__.py
- test data now corresponds to the way it is in the db.

EssKayz reviewed (approved):
Looks good.
